unity/Assets/StreamingAssets/content/MoM/ck/mom1e-tiles/tiles.ini

```diff
@@ -78,7 +77,7 @@ reverse=TileSideLaboratory
 [TileSideFrontYard]
 name={ffg:TILE_FRONT_YARD_MAD01}
 image="img/Tile_Front_Yard_MAD01"
-pps=72
```

Comment:
This is a PNG image of the correct aspect and 72 pps.  This shouldn't be changed.

Reply:
That tile is fine with this method

![works](https://user-images.githubusercontent.com/4141621/33190751-989de88a-d0b0-11e7-82ea-4db2dbe934ed.png)

Reply:
This is because of the assumption that anything with a non square aspect ratio is a large MoM tile (see other comment).  Even if we did go with the solution here, I'd still rather revert this line so that it isn't relying on hard coded magic when it doesn't need to.